lnwallet/commitment.go

```diff
@@ -310,20 +316,24 @@ func CommitScriptToRemote(chanType channeldb.ChannelType, initiator bool,
 	// we use a NUMS key to force the remote party to take a script path,
 	// with the sole tap leaf enforcing the 1 CSV delay.
 	case chanType.IsTaproot():
-		toRemoteKey, err := input.TaprootCommitScriptToRemote(
```

Comment:
btw - I dont think we use `input.TaprootCommitScriptToRemote` anywhere anymore. So can maybe remove it?